### PR TITLE
Add: EmberTemplateLintBear

### PR DESCRIPTION
**Pull request by chay2199**
Closes https://github.com/coala/coala-bears/issues/2345

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: tests/ember/EmberTemplateLintBearTest.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+bad_file = """
+<body>
+  body: {{{body}}}
+  <hr/>
+  post body
+  <div style="background-image: url({{url}})">
+</body>
+"""
+
+good_file_config = """
```
Comment (li-boxuan) [inline tests/ember/EmberTemplateLintBearTest.py]: The name is a bit confusing.

File: tests/ember/EmberTemplateLintBearTest.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import os
+
+from bears.ember.EmberTemplateLintBear import EmberTemplateLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+good_file = """
```
Comment (li-boxuan) [inline tests/ember/EmberTemplateLintBearTest.py]: Can be put into separate files under test folder.

File: tests/ember/test_files/.template-lintrc.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
```
Comment (li-boxuan) [inline tests/ember/test_files/.template-lintrc.js]: Rename the directory name, like `template-lint-test-files`

File: bears/ember/EmberTemplateLintBear.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    """
+    ember-template-lint lints templates and return error results.
+
+    This is commonly used through ember-cli-template-lint which
+    adds failing lint tests for consuming ember-cli applications.
+
+    https://github.com/ember-template-lint/ember-template-lint
+    """
+
+    LANGUAGES = {'Ember'}
```
Comment (li-boxuan) [inline bears/ember/EmberTemplateLintBear.py]: We can have a new language definition (not in this PR though) if Ember (template) is a different language from existing ones.

File: tests/ember/EmberTemplateLintBearTest.py (hunk 1 above)
Comment (chay2199) [reply tests/ember/EmberTemplateLintBearTest.py]: yes you are write, this file denotes a good file when a config file is used. Is `good_file_with_config`will be good enough?

Comment (li-boxuan) [reply tests/ember/EmberTemplateLintBearTest.py]: Yeah, that would be better

File: bears/ember/EmberTemplateLintBear.py (hunk 4 above)
Comment (chay2199) [reply bears/ember/EmberTemplateLintBear.py]: Ember is a framework of JavaScript. It is doubtful where to put it as Ember (template) only handles the template part. So one option is to put it in JS but it is totally unrelated to linting that.

Comment (jayvdb) [reply bears/ember/EmberTemplateLintBear.py]: It needs a new language definition in coala core.

**Comment by chay2199** [comment]
@jayvdb @li-boxuan I am extremely sorry, but my fork of coala-bears repo got corrupted and it no longer exists now. This PR is without a parent, please close this one and I'll make a new one for the same issue.

**Comment by jayvdb** [comment]
OK

**Pull request closed without merging**